shell/components/AppModal.vue

```diff
@@ -127,6 +140,13 @@ export default defineComponent({
         ...this.stylesPropToObj,
       };
     },
+    watcherVariableForFocusTrap(): boolean {
```

Comment:
This computed prop looks to be unused.

Reply:
You're right. Old stuff from previous iterations. Nice catch mate 🙏